setup/_pattoo/systemd.py

```diff
@@ -263,7 +263,33 @@ def preflight(config_dir, etc_dir):
         shared.log('Expected systemd directory "{}" does not exist.'.format(etc_dir))
 
 
-def run_systemd():
+def _check_symlinks(etc_dir, symlink_dir, daemons):
+    """Ensure the files in the etc dir are symlinks.
+
+    Args:
+        etc_dir: The directory that the symlinks are located in
+        symlink_dir: The directory that the symlinks point to
+        daemons: The list of system daemons
+
+    Returns:
+        None
+
+    """
+    for daemon in daemons:
+        # Initialize key variables
+        symlink_path = os.path.join(etc_dir, daemon)
+        file_path = os.path.join(symlink_dir, daemon)
+
+        # Say what we are doing
+        print('Checking if the {}.service file is a symlink '.format(daemon))
+        link = os.path.islink('{0}.service'.format(symlink_path))
+        if link is False:
```

Comment:
use the `systemctl` command to create the symlink. That way you are certain it'll be operating system compatible.

Reply:
For unittesting I might have to use os.symlink() function to set the symlinks, and test if they're present, since the systemctl command needs sudo privileges, is this okay?